_publications/jors/paper.tex

```diff
@@ -0,0 +1,362 @@
+\end{figure}
+
+\section*{Quality control}
+
+The framework has a fully automated test recipe deployed on Travis CI
+with an environment built using the Nix Package
+Manager~\cite{nix}. The environment is pinned to a specific version of
+the Nixpkgs repository ensuring fully reproducible build and test
+phases as well as ensuring that the development and automated testing
+environments are identical. The full test recipe is outlined in a
+\texttt{.travis.yml} file stored in the repository~\cite{travisyml}
+and consists of the following steps.
+
+\begin{itemize}
+  \item Build the Nix environment from a cached storage reducing the
+    build time.
+  \item Run automated tests on Jupyter Notebooks using
+    NBval~\cite{nbval} and Py.test~\cite{pytest}.
+  \item Run validation tests on HTML files using
+    HTMLProofer~\cite{htmlproofer}.
+  \item Lint and test front-end Coffeescript using appropriate tools.
+  \item Display a temporary version of the website using
+    Surge~\cite{surge} for visual review.
+\end{itemize}
+
+\section*{(2) Availability}
+\vspace{0.5cm}
+\section*{Operating system}
+
+The PFHub framework can be deployed on any platform supporting Nix,
+which includes all contemporary Linux and Mac OS X platforms. Since
+the framework is built with Jekyll and automated front-end processing,
+it can be deployed on GitHub's Pages infrastructure, which enables
+streamlined deployment without the need for any back-end
+infrastructure.
+
+\section*{Programming language}
+
+PFHub is currently built and tested using the programming languages
+and versions outlined in Table~\ref{tab:versions}.
+
+\begin{table}[h!]
+  \centering
+  \caption{PFHub programming languages and corresponding supported
+    versions.}
+  \begin{tabular}{|l|l|}
+    \hline
+    Language         & Version \\
+    \hline
+    HTML             & 5       \\
+    Jupyter Notebook & 5.4.0   \\
+    JavaScript       & 5       \\
+    Nix              & 2.1.3   \\
+    CoffeeScript     & 1.12.7  \\
+    CSS              & 4       \\
+    \hline
+  \end{tabular}
+  \label{tab:versions}
+\end{table}
+
+
+\section*{Additional system requirements}
+
+There are no additional system requirements.
+
+\section*{Dependencies}
+
+The entire environment can be built using the Nix Package Manager so
+the only required dependency is a functional Nix installation. The
+PFHub framework has over 2000 separate package dependencies using
+data from the Nix package manager. The full dependency graph for PFHub
+can be seen online~\cite{dependencies}.
+
+\section*{List of contributors}
+
+1. Wheeler, Daniel; A, \githublink{wd15} \\
+2. Keller, Trevor; A, \githublink{tkphd} \\
+3. DeWitt, Stephen J.; E, \githublink{stvdwtt} \\
+4. Jokisaari, Andrea M.; D, \githublink{amjokisaari} \\
+5. Schwen, Daniel; D, \githublink{dschwen} \\
+6. Guyer, Jonathan E.; A, \githublink{guyer} \\
+
+See the contributors list on GitHub~\cite{contributors}.
+
+\section*{Software location:}
+
+{\bf Archive}
```

Comment:
What is this an Archive of?  Data?

Reply:
Again, this is their crazy TeX template. We have to use their section headings. Parts of this are basically filling in a form.